.github/workflows/validate-commit-message.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+
+# requires: gitpython
+
+import os
+from argparse import ArgumentParser
+
+from git import Repo
+
+KEYWORDS = ['nopackage', 'deploy', 'docs']
+EXISTING_PACKAGES = []
+
+
+def prefix_invalid(prefix):
```

Comment:
This could still include the `splitlines` and `split(':')`, both can be applied to either the title or commit message